common/model/bundle.c

```diff
@@ -15,15 +19,75 @@
 #define NUMBER_OF_SECURITY_LEVELS 3
 #define NORMALIZED_FRAGMENT_LENGTH 27
 
-void normalized_bundle(trit_t const *const bundle_hash,
-                       byte_t *const normalized_bundle_hash) {
+static UT_icd bundle_transactions_icd = {sizeof(struct _iota_transaction), 0, 0,
```

Comment:
This duplicates the stuff @ifullgaz did for the transfers calculation etc. Can't we share some code here at least?

Reply:
Well aware, we can share code here,  i was reluctant to use the transfer version because that demands creating a loading functions for `transfer_t` from storage, they would have been a little ugly because you would have to do some queries for knowing the transfer types and allocate it therefor, in addition, transfer `OUT` contains `seed` so i imagined it's purpose is in generating a bundle rather then reading it from transaction's trytes..

Reply:
Thanks for #294 !